x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/empty_state.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPanel, EuiEmptyPrompt, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { DOCS_PREFIX } from '../../../routes';
+
+export const EmptyState: React.FC = () => {
+  return (
+    <EuiPanel color="subdued">
+      <EuiEmptyPrompt
+        iconType="gear"
```

Comment:
@daveyholler Also feel free to shout if you have feedback on the gear icon for the schema page, I know we also use the gear icon on the Result Settings empty state 🤔

Reply:
I'd go with the `database` icon here instead.

Reply:
Awesome!! Let's do it 😎 
```suggestion
        iconType="database"
```